src-electron/db/query-endpoint-type.js

```diff
@@ -165,14 +165,16 @@ async function selectEndpointType(db, id) {
 async function selectAllClustersDetailsFromEndpointTypes(
   db,
   endpointTypes,
-  options = null
```

Comment:
Passing options allows other flags coming from the handlebar helper. This helps in not updating the function signature.

Reply:
The intent is valid. But the options were passed in a handlebars specific way (options.hash) and I needed to call this functions out of template context. I don't want db layer to be polluted by handlebars specifics.